### ISSUE TITLE
The extension is not working

### ISSUE THREAD

**Issue by std4453** [gwwar/z-context#25]
Browser: Chrome 104.0.5112.79
System: macOS Big Sur

The z-index tab is always blank:
<img width="522" alt="image" src="https://user-images.githubusercontent.com/13191718/184078586-cd7d1774-6c44-47f4-8120-311582de02b5.png">

Judging from Chrome WebStore comments, the extension stopped working at around August 2021. What's causing this? Is this related to #19 ? Anything I can do to help?

**Comment by nikicat** [comment]
It works for me on Chromium Version 105.0.5195.102 (Official Build)

**Comment by funkadelic** [comment]
Did _not_ work for me either on Chrome 105.0.5195.102 (OS X 12.5.1, MacBook Pro (16-inch, 2021))

**Comment by nikicat** [comment]
Yup, on Google Chrome Version 105.0.5195.102 (Official Build) (64-bit) I do not even see the panel

**Comment by jxmot** [comment]
Not working in Version 108.0.5359.71 (Official Build) (64-bit), the panel is there but always empty.

**Comment by ezdub** [comment]
I have found that z-context v2.1.0 doesn't work (empty panel) upon first install until the browser is restarted.  Still looking into why but I suspect one of the listeners is not connecting.
Temp fix: restart chrome browser after install.

**Comment by PerpetualWar** [comment]
Same here... also do not work after browser restart. Im using brave browser though, but not working.

**Comment by gwwar** [comment]
I've updated to v3 https://github.com/gwwar/z-context/pull/29, and the new version should be available from https://chromewebstore.google.com/detail/z-context/jigamimbjojkdgnlldajknogfgncplbh

Let me know if y'all are still seeing issues in 3.0
